cmd/keycmd/transfer.go

```diff
@@ -732,6 +763,13 @@ func exportFromP(
 		}
 		return err
 	}
+	pContext := wallet.P().Builder().Context()
+	pFeeCalculator := avagofee.NewDynamicCalculator(pContext.ComplexityWeights, pContext.GasPrice)
+	txFee, err := pFeeCalculator.CalculateFee(unsignedTx)
+	if err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser("P-Chain Paid fee: %.9f", float64(txFee)/float64(units.Avax))
```

Comment:
its great that we are printing which kind of fee is being paid (p chain / c chain, etc)